composer.json

```diff
@@ -226,9 +227,9 @@
         "google/cloud-workflows": "0.4.2",
         "google/grafeas": "0.8.3",
         "google/longrunning": "0.2.6",
+        "google/shopping-common-protos": "0.2.0",
         "google/shopping-merchant-inventories": "0.1.1",
-        "google/shopping-merchant-reports": "0.3.0",
-        "google/shopping-common-protos": "0.2.0"
+        "google/shopping-merchant-reports": "0.3.0"
```

Comment:
Not really sure why this happened. The order just shuffled.

Reply:
It's because the script which writes it to the JSON file reorders some things if they were added manually.